fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java

```diff
@@ -128,7 +128,21 @@ private static void init() {
         commandStrategies.put(
                 CommandContext.resource("loans\\/external-id\\/" + UUID_PARAM_REGEX + OPTIONAL_QUERY_PARAM_REGEX).method(GET).build(),
                 "getLoanByExternalIdCommandStrategy");
+        commandStrategies.put(CommandContext.resource("savingsaccounts\\/" + NUMBER_REGEX + OPTIONAL_QUERY_PARAM_REGEX).method(GET).build(),
+                "getSavingsAccountByIdCommandStrategy");
         commandStrategies.put(CommandContext.resource("savingsaccounts").method(POST).build(), "applySavingsCommandStrategy");
+        commandStrategies.put(
+                CommandContext.resource("savingsaccounts\\/" + NUMBER_REGEX + "\\/transactions\\?command=deposit").method(POST).build(),
+                "depositSavingsAccountCommandStrategy");
+        commandStrategies.put(
+                CommandContext.resource("savingsaccounts\\/" + NUMBER_REGEX + "\\/transactions\\?command=withdrawal").method(POST).build(),
+                "withdrawalSavingsAccountCommandStrategy");
+        commandStrategies.put(
+                CommandContext.resource("savingsaccounts\\/" + NUMBER_REGEX + "\\/transactions\\?command=holdAmount").method(POST).build(),
+                "holdAmountSavingsAccountCommandStrategy");
+        commandStrategies.put(CommandContext
```

Comment:
Similarly as above having a generic one probably would be beneficial:
`SavingsAccountAdjustTransactionCommandStrategy`

Reply:
fixed